components/BWSHead/index.js

```diff
@@ -24,33 +24,43 @@ class BWSHead extends React.Component {
           <title>{pageTitle}</title>
 
           <meta charSet="utf-8" />
+          <meta name='application-name' content='BWS Collection' />
+          <meta name="referrer" content="origin-when-cross-origin" />
 
-          <meta
-            name="viewport"
-            content="width=device-width, initial-scale=1, shrink-to-fit=no"
-          />
+          <meta name='apple-mobile-web-app-capable' content='yes' />
+          <meta name='apple-mobile-web-app-status-bar-style' content='default' />
+          <meta name='apple-mobile-web-app-title' content='BWS Collection' />
+          <meta name='description' content={pageDescription || defaultDescription}/>
+          <meta name='format-detection' content='telephone=no' />
+          <meta name='mobile-web-app-capable' content='yes' />
 
-          <meta name="referrer" content="origin-when-cross-origin" />
-          <meta
-            name="description"
-            content={pageDescription || defaultDescription}
-          />
-
-          <meta name="og:site_name" content={defaultPageTitle} />
-          <meta name="og:title" content={pageTitle || defaultPageTitle} />
-          <meta name="og:type" content={seoType || "website"} />
-          <meta name="og:image" content={pageImage || defaultImageUrl} />
```

Comment:
Looks like this tag was dropped

Reply:
I can't tell which tag you're referring to. Looks like this comment is in the middle of a block of deleted lines